cms/templates/js/no-group-configurations.underscore

```diff
@@ -1,3 +1,3 @@
 <div class="no-group-configurations-content">
-    <p><%= gettext("You haven't created any group configurations yet.") %><a href="#" class="button new-button"><i class="icon-plus"></i><%= gettext("Add your first Group Configuration") %></a></p>
+    <p><%= gettext("You haven't created any group configurations yet.") %><a href="#" class="button new-button"><i class="icon-plus"></i><%= gettext("Add your first Experiment Configuration") %></a></p>
```

Comment:
@catong, hmm, you're stronger in the Grammar Force than I, Master Jedi. Unless these are formal terms or proper nouns, I think we can lose the caps mid-sentence. Sound okay?